packages/shopify-app-session-storage-postgresql/src/migrations.ts

```diff
@@ -2,12 +2,60 @@ import {PostgresConnection} from './postgres-connection';
 
 export const migrationMap = new Map([
   ['migrateScopeFieldToVarchar1024', migrateScopeFieldToVarchar1024],
+  ['migrateToCaseSensitivity', migrateToCaseSensitivity],
 ]);
 
-// need change the sizr of the scope column from 255 to 1024 char
+// need change the size of the scope column from 255 to 1024 char
 export async function migrateScopeFieldToVarchar1024(
   connection: PostgresConnection,
 ): Promise<void> {
-  await connection.query(`ALTER TABLE ${connection.sessionDBIdentifier} 
-    ALTER COLUMN scope TYPE varchar(1024)`);
+  await connection.query(`ALTER TABLE "${connection.sessionDBIdentifier}"
+    ALTER COLUMN "scope" TYPE varchar(1024)`);
+}
+
+export async function migrateToCaseSensitivity(
```

Comment:
Should this be done within a transaction to avoid data loss?
Comment made from @paulomarg on my `sqlite` migration :)